data/tribes/buildings/productionsites/amazons/rare_tree_cutters_hut/init.lua

```diff
@@ -32,6 +32,7 @@ descriptions:new_productionsite_type {
             "call=harvest_balsa",
             "call=harvest_rubber",
             "call=harvest_ironwood",
+            "return=skipped"
```

Comment:
amazons can't get more experience in the rare treecutters hut as it is operated by a master only.